nengo_viz/static/viz_slider.js

```diff
@@ -151,6 +150,13 @@ VIZ.Slider.prototype = Object.create(VIZ.Component.prototype);
 VIZ.Slider.prototype.constructor = VIZ.Slider;
 
 VIZ.Slider.prototype.set_value = function(slider_index, value, immediate) {
+    console.assert(typeof slider_index == 'number');
+    console.assert(typeof value == 'number');
+    console.assert(typeof immediate == 'boolean');
```

Comment:
Cool...  I could see getting into the habit of including those...  it'll be a bit of a black art to figure out when/where to do that.